spring-boot/admin/src/main/resources/application.yml

```diff
@@ -3,29 +3,105 @@
 
 mvn.pom.project.version: @@project.version@@
 
-# user and password are expected to be for a gmail account
-ygg.mail.verification.server.url: http://localhost:8080
-
 logging:
   level:
     org.springframework.security: INFO
     nl.corizon.ygg: DEBUG
 
+
+
 # Necessary in order to run offline     
-ygg.admin.database.insertSampleDataOnStartup: true
-ygg.admin.gettingStarted.consumer.testService.id: 1
-ygg.admin.gitService.url: http://localhost
-ygg.admin.gitService.user: user
-ygg.admin.gitService.password: password
-ygg.admin.api-base-url-for-gateway: http://localhost
-ygg.proxy.executable-jar.path: path
-ygg.proxy.executable-jar.filename-original-prefix: prefix
-ygg.proxy.executable-jar.filename-custom.prefix: prefix
-ygg.proxy.executable-jar.filename-custom.suffix: suffix
-ygg.proxy.generate-bootstrap-properties.spring.application.name.prefix: prefix
-ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri: http://localhost
+ygg:
+
+# user and password are expected to be for a gmail account
+
+   mail:
+      server:
+         url: "smtp.serviciodecorreo.es"
+         port: 465
+      sender:
+         from: "coatrack@kyoe.es"
+         user: ""
+         password: ""
+      verification.server.url: http://localhost:8080
+
+   admin:
+      database.insertSampleDataOnStartup: true
+      server:
+         url: http://localhost:8080
+      servicecovers:
+         path: /tmp
+         url: http://localhost/coatrack-servicecovers/
+      payment.commision:
+         variable: 50
+         fix: 1
+         bookKeeping.contact: fperezduran@bebr.nl
+      call-request-fee: 0
+    
+      gettingStarted.consumer.testService.id: 1
+      gitService:
+          url: http://localhost
+          user: user
+          password: password
+      api-base-url-for-gateway: http://localhost
+
+      gettingStarted:
+         consumer:
+           testService:
+# the following two elements are required to load the test service for getting started wizard
+                provider.username: BrunoSilvaATB
+                uriIdentifier: getting-started-service-consumer
+   proxy:
+      executable-jar:
+         generated.path: /tmp
+         template.url: https://www.kyoe.es/proxies/coatrack-proxy-2.0.0-SNAPSHOT.jar
```

Comment:
draft

Reply:
If we merge the current version, it will not work at runtime. This is why I would suggest to add an explanation and put only the prefix.
```suggestion
         # URL of the "template jar" that CoatRack uses as basis to generate proxies (without the trailing version number and .jar filename extension)
         template.url: https://www.kyoe.es/proxies/coatrack-proxy-
```